releasenotes/notes/grafana-opensearch-datasource-configuration-04202c059f1abd05.yaml

```diff
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
+    Updates the default Grafana OpenSearch datasource configuration to use
+    values for OpenSearch that work out of the box. Replaces the Elasticsearch
+    values that were previously being used. The new configuration can be
+    applied by deleting your datasource and reconfiguring Grafana through kolla
```

Comment:
What happens if you don't delete the datasource?

Reply:
you just wont have working opensearch metrics